charts/kubescape-cloud-operator/templates/kubescape/deployment.yaml

```diff
@@ -99,6 +102,12 @@ spec:
           value: "{{ .Values.cloudProviderMetadata.gkeProject }}"
         {{- end -}}
         {{- end }}
+        {{- if .Values.otelCollector.enabled }}
+        - name: ACCOUNT_ID
```

Comment:
[TODO]: Reconsider adding the account ID here since it is mounted via the configMap